src/components/Chart.jsx

```diff
@@ -110,6 +110,7 @@ class Chart extends Component {
                 <DrawToolsSettingsDialog />
                 <AggregateChartSettingsDialog />
                 <StudySettingsDialog />
+                <div id="drawing-mouse-pointer" className="drawing-mouse-pointer" />
```

Comment:
don't use `id`. SmartCharts need to support multiple chart instances in the very near future